src/content/apis/travel/IRCTC_API.md

```diff
@@ -0,0 +1,10 @@
+---
+name: IRCTC API
+description: An API with various functionalities for Indian railways IRCTC API. We are not affiliated with Indian railways IRCTC in any form, and this is not an official API, but all the results are real-time and accurate.
+categories:
+  - Travel
+features: Get real-time and accurate result.
+  https: true
+url: https://irctc1.p.rapidapi.com/api/v1/searchStation
```

Comment:
Does the API have any proper documentation?
The current URL you shared doesn't have any proper info about how to use it.

Reply:
Should I put the link of documentation also ??

Reply:
Instead of this link itself, I would suggest adding the documentation link 🙏

Reply:
OK 😊